src/objective-c/examples/BazelBuildSamples/ios-sample/ios-sample.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,413 @@
+// !$*UTF8*$!
```

Comment:
And also `xcodeproj` is not desirable too (since bazel does not require this, even though Tulsi can generate it)